eng/common/scripts/Verify-Links.ps1

```diff
@@ -74,6 +75,8 @@ param (
   [string] $requestTimeoutSec  = 15
 )
 
+Set-StrictMode -Version 3.0
```

Comment:
Please do a little more testing given you are setting this now as it can break scripts in random ways.